orttraining/orttraining/models/runner/pipeline.cc

```diff
@@ -38,16 +38,18 @@ bool Slot::IsBackward() const {
 
 void Slot::Show() const {
     const int waited_event_id = waited_events.size() > 0 ? waited_events[0] : -1;
-    const int recorded_event_id = recorded_events.size() > 0 ? recorded_events[0] : -1;
+    const int waited_event_id_after_recv = waited_events.size() > 0 ? waited_events[1] : -1;
+    const int recorded_event_id_before_send = recorded_events.size() > 0 ? recorded_events[0] : -1;
+    const int recorded_event_id = recorded_events.size() > 0 ? recorded_events[1] : -1;
     switch (type) {
     case Empty:
-        std::cout << "         ";
+        std::cout << "             ";
         break;
     case Forward:
-        std::cout << " F" << batch_id << "(" << waited_event_id << "," << recorded_event_id << ")" << " ";
+        std::cout << " F" << batch_id << "(" << waited_event_id << "," << waited_event_id_after_recv << "," << recorded_event_id_before_send << "," << recorded_event_id << ")" << " ";
```

Comment:
>std::cout << " F" << batch_id << "(" << waited_event_id << "," << waited_event_id_after_recv << "," << recorded_event_id_before_send << "," << recorded_event_id << ")" << " "; [](start = 8, length = 175)

Nit: line wrap

Reply:
same as line 52

---
In reply to: [425485582](https://github.com/microsoft/onnxruntime/pull/3823#discussion_r425485582) [](ancestors = 425485582)

Reply:
I also auto-format the entire document.

---
In reply to: [425485721](https://github.com/microsoft/onnxruntime/pull/3823#discussion_r425485721) [](ancestors = 425485721,425485582)